eWonConnector-gateway/src/main/java/org/imdc/ewon/config/EwonConnectorSettings.java

```diff
@@ -36,11 +38,13 @@
 
 	public static final Category MainCategory = new Category("EwonConnectorSettings.Category.Main", 10).include(Name, Enabled, PollRate, PollRateUnits);
 	public static final Category AccountCategory = new Category("EwonConnectorSettings.Category.Account", 50).include(Account, UserName, Password, APIKey);
+	public static final Category DeviceCategory = new Category("EwonConnectorSettings.Category.Device", 75).include(EwonUserName, EwonPassword);
```

Comment:
Official java code style says that final variables/constants should have their name in all caps, and spaces filled with an underscore, for example DEVICE_CATEGORY.